docs/getting_started/installation_guide/platforms/karamel.rst

```diff
@@ -0,0 +1,87 @@
+.. _karamel-installer:
+
+=======================
+Karamel
+=======================
+
+#. Download and install `Karamel <http://www.karamel.io/>`_ 
+#. Run Karamel.
+#. Click on the "Load Cluster Definition" menu item in Karamel. You are now prompted to select a cluster definition YAML file.
+
+Example cluster definitions can be found on `github <https://github.com/logicalclocks/karamel-chef/tree/master/cluster-defns>`_.
+
+
+**Choosing which services to run on which nodes**
+
+You need to decide which services you will install on which nodes. In Karamel, we design a set of *Node Groups*, where each *Node Group* defines a stack of services to be installed on a machine. Each machine will only have one *Node Group* set of services.
```

Comment:
Should maybe say "Each machine will only be part of one *Node Group*" instead.

Reply:
That's better, fixed.